dev-env/README.md

```diff
@@ -1,17 +1,44 @@
 # dev-env
 
 This directory contains supporting files for the included MetalLB development
-environment. The environment is run using the following command:
+environment. The environment is run using the following command from the root of your git clone, which runs MetalLB in an unconfigured state:
 
 ```
 inv dev-env
 ```
 
-There is also support for configuring MetalLB to peer with a BGP router running
+For configuring MetalLB to peer with a BGP router running
 in a container:
 
 ```
 inv dev-env --protocol bgp
 ```
 
 For more information, see [the BGP dev-env README](bgp/README.md).
+
+You may also launch a dev environment with layer2, with:
+
+```
+inv dev-env --protocol layer2
+```
+
+For more information, see help:
+
+```
+inv dev-env --help
+```
+
+## Requirements
+
+* Go 1.15+
+* Python 3
+* [KIND - Kubernetes in Docker](https://kind.sigs.k8s.io/docs/user/quick-start/)
+  * And optionally [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
```

Comment:
Is this optional? I just happened to be installing from scratch and I noticed that I needed it.

Reply:
changed.

Reply:
Thanks